manifests/art.yaml

```diff
@@ -0,0 +1,16 @@
+updates:
+  - file: "{MAJOR}.{MINOR}/cluster-logging.v{MAJOR}.{MINOR}.0.clusterserviceversion.yaml" # relative to this file
+    update_list:
+    # replace metadata.name value
+    - search: "clusterlogging.v{MAJOR}.{MINOR}.0"
```

Comment:
@ewolinetz fixed to match https://github.com/openshift/cluster-logging-operator/pull/204/files#diff-5766fe294b4d426421892b30ee5365eaR6

thanks